app/controllers/api/v1/memberships_controller.rb

```diff
@@ -4,9 +4,18 @@ class Api::V1::MembershipsController < Api::ApiController
   resource_actions :index, :show, :create, :update, :deactivate
   schema_type :strong_params
 
-  allowed_params :create, links: [:user, :user_group]
```

Comment:
Note: this is safe because ActiveInteraction whitelists arguments to only the filters (eg `string :join_token`) specified by the operation.